tools/fuzz/fuzz.cpp

```diff
@@ -39,6 +39,8 @@
 
 namespace {
 
+enum class FuzzingTarget { kSPIR_V, kWGSL };
```

Comment:
I think the preferred Google style convention would be to call these kSpirV and kWgsl.

Reply:
Fixed. I went with `kSpirv` as in `SpirvTools`.